layers/+tools/transmission/packages.el

```diff
@@ -29,50 +29,94 @@
     :defer t
     :init (progn
             (spacemacs/set-leader-keys "att" 'transmission)
+
             (spacemacs/declare-prefix-for-mode 'transmission-mode "ma" "add")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "mg" "goto")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "ms" "set")
             (spacemacs/declare-prefix-for-mode 'transmission-mode "mt" "toggle")
+            (spacemacs/declare-prefix-for-mode 'transmission-mode "mQ" "Queue")
             (spacemacs/set-leader-keys-for-major-mode 'transmission-mode
-              "gt" 'transmission-files
-              "at" 'transmission-add
-              "sd" 'transmission-set-download
-              "gp" 'transmission-peers
-              "gi" 'transmission-info
-              "sl" 'transmission-set-ratio
+              "D"  'transmission-delete
               "m"  'transmission-move
-              "r"  'transmission-remove
-              "ts" 'transmission-toggle
+              "q"  'transmission-quit
+              "R"  'transmission-remove
+              "v"  'transmission-verify
+              "S"  'tabulated-list-sort
+
+              ;; add
               "aa" 'transmission-trackers-add
```

Comment:
"aa" and "at" are confusing to me..

Why not `aa` for `transmission-add` and `at` for `transmission-trackers-add`?

Reply:
To be honest, I agree with you here. These bindings were already present from before and I believe the author meant for it to match the binding for `transmission-tracker-add` in `transmission-info-mode`. It would make more sense to change them right?